### ISSUE TITLE
Postgres migration script appears to be broken (again)

### ISSUE THREAD

**Issue by turt2live** [matrix-org/synapse#2565]
### Description

```
2017-10-22 02:08:45,904 - synapse_port_db - 546 - ERROR -
Traceback (most recent call last):
  File "/home/travis/.synapse/bin/synapse_port_db", line 539, in run
    consumeErrors=True,
FirstError: FirstError[#9, [Failure instance: Traceback: <type 'exceptions.AttributeError'>: type object 'Store' has no attribute '_simple_update_txn'
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:500:errback
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:567:_startRunCallbacks
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:653:_runCallbacks
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:1442:gotResult
--- <exception caught here> ---
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/internet/defer.py:1384:_inlineCallbacks
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/failure.py:408:throwExceptionIntoGenerator
/home/travis/.synapse/bin/synapse_port_db:338:handle_table
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/threadpool.py:250:inContext
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/threadpool.py:266:<lambda>
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/context.py:122:callWithContext
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/python/context.py:85:callWithContext
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/enterprise/adbapi.py:307:_runWithConnection
/home/travis/.synapse/local/lib/python2.7/site-packages/twisted/enterprise/adbapi.py:298:_runWithConnection
/home/travis/.synapse/bin/synapse_port_db:126:r
/home/travis/.synapse/bin/synapse_port_db:334:insert
/home/travis/.synapse/local/lib/python2.7/site-packages/synapse/storage/_base.py:800:_simple_update_one_txn
]]
```

This is on a dev environment, which might be part of the problem. Server is based off of the develop branch.

### Version information

- **Homeserver**: self-hosted dev synapse
- **Version**:        0.24.0-rc1 (develop branch)
- **Install method**: pip
- **Platform**:    Ubuntu 16.04 VM

**Comment by Josue-T** [comment]
Hello,
Same problem !

**Comment by richvdh** [comment]
For reference this got broken here: https://github.com/matrix-org/synapse/pull/2429/files#diff-13caa7216e6c08703de17985ecd65997R800

**Comment by ara4n** [comment]
now fixed again.

**Comment by steliosmilidonis** [comment]
I have the following issue migrating from sqlite to postgres

2017-11-17 13:46:15,081 - synapse.metrics - 162 - INFO - Collecting gc 0
2017-11-17 13:46:15,089 - synapse.metrics - 162 - INFO - Collecting gc 0
2017-11-17 13:46:15,097 - synapse.metrics - 162 - INFO - Collecting gc 0
2017-11-17 13:46:15,102 - synapse.metrics - 162 - INFO - Collecting gc 0
users_in_pubic_room: 100% (1614/1614)
user_directory_search: 100% (1616/1616)
users_who_share_rooms: 100% (1639/1639)
user_directory: 100% (1616/1616)
2017-11-17 13:46:15,146 - synapse.metrics - 162 - INFO - Collecting gc 0
2017-11-17 13:46:15,147 - synapse_port_db - 546 - ERROR - 
Traceback (most recent call last):
  File "/usr/bin/synapse_port_db", line 539, in run
    consumeErrors=True,
FirstError: FirstError[#4, [Failure instance: Traceback: <type 'exceptions.AttributeError'>: type object 'Store' has no attribute '_simple_update_txn'
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:498:errback
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:565:_startRunCallbacks
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:651:_runCallbacks
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:1355:gotResult
--- <exception caught here> ---
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:1297:_inlineCallbacks
/usr/lib/python2.7/dist-packages/twisted/python/failure.py:389:throwExceptionIntoGenerator
/usr/bin/synapse_port_db:338:handle_table
/usr/lib/python2.7/dist-packages/twisted/python/threadpool.py:246:inContext
/usr/lib/python2.7/dist-packages/twisted/python/threadpool.py:262:<lambda>
/usr/lib/python2.7/dist-packages/twisted/python/context.py:118:callWithContext
/usr/lib/python2.7/dist-packages/twisted/python/context.py:81:callWithContext
/usr/lib/python2.7/dist-packages/twisted/enterprise/adbapi.py:307:_runWithConnection
/usr/lib/python2.7/dist-packages/twisted/enterprise/adbapi.py:298:_runWithConnection
/usr/bin/synapse_port_db:126:r
/usr/bin/synapse_port_db:334:insert
/usr/lib/python2.7/dist-packages/synapse/storage/_base.py:800:_simple_update_one_txn
]]
2017-11-17 13:46:15,187 - synapse.metrics - 162 - INFO - Collecting gc 0
Traceback (most recent call last):
  File "/usr/bin/synapse_port_db", line 539, in run
    consumeErrors=True,
FirstError: FirstError[#4, [Failure instance: Traceback: <type 'exceptions.AttributeError'>: type object 'Store' has no attribute '_simple_update_txn'
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:498:errback
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:565:_startRunCallbacks
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:651:_runCallbacks
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:1355:gotResult
--- <exception caught here> ---
/usr/lib/python2.7/dist-packages/twisted/internet/defer.py:1297:_inlineCallbacks
/usr/lib/python2.7/dist-packages/twisted/python/failure.py:389:throwExceptionIntoGenerator
/usr/bin/synapse_port_db:338:handle_table
/usr/lib/python2.7/dist-packages/twisted/python/threadpool.py:246:inContext
/usr/lib/python2.7/dist-packages/twisted/python/threadpool.py:262:<lambda>
/usr/lib/python2.7/dist-packages/twisted/python/context.py:118:callWithContext
/usr/lib/python2.7/dist-packages/twisted/python/context.py:81:callWithContext
/usr/lib/python2.7/dist-packages/twisted/enterprise/adbapi.py:307:_runWithConnection
/usr/lib/python2.7/dist-packages/twisted/enterprise/adbapi.py:298:_runWithConnection
/usr/bin/synapse_port_db:126:r
/usr/bin/synapse_port_db:334:insert
/usr/lib/python2.7/dist-packages/synapse/storage/_base.py:800:_simple_update_one_txn
]]

**Comment by richvdh** [comment]
For the record, this was broken in v0.24.0-rc1 and fixed in v0.25-rc1
